data/data/aws/vpc/common.tf

```diff
@@ -17,10 +17,8 @@ locals {
   vpc_id = "${aws_vpc.new_vpc.id}"
 
   // When referencing the _ids arrays or data source arrays via count = , always use the *_count variable rather than taking the length of the list
-  worker_subnet_ids   = "${aws_subnet.worker_subnet.*.id}"
-  master_subnet_ids   = "${aws_subnet.master_subnet.*.id}"
-  worker_subnet_count = "${local.new_az_count}"
-  master_subnet_count = "${local.new_az_count}"
+  master_subnet_ids      = "${aws_subnet.master_subnet.*.id}"
+  master_az_to_subnet_id = "${zipmap(local.new_subnet_azs, local.master_subnet_ids)}"
```

Comment:
Trying to hold this in my head at the same time as #1045 moving all machines into a private subnet is melting my brain :p.  @crawford, do you have a feeling about how close we are to being able to [pull the hold][1] from #1045?

[1]: https://github.com/openshift/installer/pull/1045#issuecomment-459137689